.github/workflows/benchmarks.yml

```diff
@@ -0,0 +1,55 @@
+name: Benchmarks
+
+on: [pull_request]
+
+jobs:
+  benchmarks:
+    name: benchmarks
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+
+      - name: Initialize tachometer comment
```

Comment:
What does "Initialize tachometer comment" mean? Is "comment" supposed to be there?

Reply:
Got it from here: https://github.com/andrewiggins/tachometer-reporter-action#multiple-benchmark-jobs and https://github.com/andrewiggins/tachometer-reporter-action#initialize

But reading it again, looks like it's only required if we're doing multiple jobs in parallel (which is something we could do, but I was starting with this simpler config). Removed for now.

Reply:
Yup, that's right. If you start running benchmarks in different jobs then you'll want an initial job to create the PR comment the other jobs will then fill in (there is a way to do it without needing an initial job but it is susceptible to race conditions leading to multiple result comments so I just recommend the "initialize" job approach. )